src/App.js

```diff
@@ -1,16 +1,84 @@
 import React from 'react';
 import './App.css';
+import ChatEntry from './components/ChatEntry';
+import TimeStamp from './components/TimeStamp';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+const LOG = [
+  {
+    id: 1,
+    sender: 'Vladimir',
+    body: 'why are you arguing with me',
+    timeStamp: '2018-05-29T22:49:06+00:00',
+  },
+  {
+    id: 2,
+    sender: 'Estragon',
+    body: 'Because you are wrong.',
+    timeStamp: '2018-05-29T22:49:33+00:00',
+  },
+  {
+    id: 3,
+    sender: 'Vladimir',
+    body: 'because I am what',
+    timeStamp: '2018-05-29T22:50:22+00:00',
+  },
+  {
+    id: 4,
+    sender: 'Estragon',
+    body: 'A robot.',
+    timeStamp: '2018-05-29T22:52:21+00:00',
+  },
+  {
+    id: 5,
+    sender: 'Vladimir',
+    body: 'Notabot',
+    timeStamp: '2019-07-23T22:52:21+00:00',
+  },
+];
```

Comment:
This is already in a separate file that is imported. Did you pull it in here so you could remember how the data structure looked? If so, that's just fine! We should get rid of it, though, for the final submission.
```suggestion

```